bpf/net/bpf_filter.c

```diff
@@ -623,19 +437,6 @@ bpf_validate(const struct bpf_insn *f, int len)
 			case BPF_ABS:
 			case BPF_IND:
 			case BPF_MSH:
-				/*
```

Comment:
I'd put the comment about there being no maximum packet data size back, to indicate why we're not checking the offset field in those instructions.